cmd/util/utils.go

```diff
@@ -24,39 +24,51 @@ var testNet3GenHash = Hash{
 	0x01, 0xea, 0x33, 0x09, 0x00, 0x00, 0x00, 0x00,
 }
 
+var regTestGenHash = Hash{
+  0x06, 0x22, 0x6e, 0x46, 0x11, 0x1a, 0x0b, 0x59,
+  0xca, 0xaf, 0x12, 0x60, 0x43, 0xeb, 0x5b, 0xbf,
+  0x28, 0xc3, 0x4f, 0x3a, 0x5e, 0x33, 0x2a, 0x1f,
+  0xc7, 0xb2, 0xb7, 0x3c, 0xf1, 0x88, 0x91, 0x0f,
+}
+
+func GenHashForNet(net wire.BitcoinNet) Hash {
+	switch net {
+		case wire.TestNet3:
+			return testNet3GenHash
+		case wire.MainNet:
+			return mainNetGenHash
+		case wire.TestNet: // yes, this is regtest
+			return regTestGenHash
+	}
+	panic("net not supported by GenHashForNet")
```

Comment:
I think returning and error here is better. In some parts of the code, you'll see functions with ```panic(err)``` instead of ```return err``` and this was mostly for debugging since with panics, you get the line # of where it errored out.

I think something like
``` func GenHashForNet(net wire.BitcoinNet) (Hash, error)```
and
``` return nil, fmt.Errorf("net not supported\n")```
is good